cucumber-rails.gemspec

```diff
@@ -12,23 +12,23 @@ Gem::Specification.new do |s|
 
   s.license = 'MIT'
 
-  s.add_runtime_dependency('capybara', ['>= 1.1.2', '< 3'])
+  s.add_runtime_dependency('capybara', ['>= 1.1.2', '< 4'])
```

Comment:
This also resolves @Kosmas' request here: https://github.com/cucumber/cucumber-rails/issues/375#issuecomment-380043499 :+1:

Reply:
:+1: